doc/processing.md

```diff
@@ -0,0 +1,92 @@
+## How OAR Facilities Are Created And Matched
+
+The list of searchable facilities in the Open Apparel Registry is compiled from
+contributor submitted lists. The application uses statistical text analysis to
+compare newly submitted items to the existing list of facilities in an attempt
+to automatically match highly similar items and to suggest potential matches
+that require confirmation when there weaker similarities between items.
+
+Contributors submit their facility lists to the Open Apparel registry for
+processing by uploading lists in comma-separated value (CSV) format. All popular
+spreadsheet programs, including Microsoft Excel, can save a sheet in CSV format.
+The Open Apparel Registry provides a downloadable template, but any CSV can be
+read and processed if it meets the following criteria:
+
+  - Includes a header row with the following field names (in lower case)
+    - country
+    - name
+    - address
+  - Less than 5MB in size.
+
+Each line in a contributed facility list CSV file is processed separately and
+passes through three steps:
+
+  - _Parse_: The line is separated into distinct country, name, and address fields
```

Comment:
Consider making these into `###` subheadings, which I believe would make them linkable on github.

Reply:
Fixed in 4d2c0ab